submissions/asaMitaka/dom/script.js

```diff
@@ -0,0 +1,232 @@
+        alt: 'Ye',
+        source: `
+            <iframe style="border-radius:12px" src="https://open.spotify.com/embed/album/2Ek1q2haOnxVqhvVKqMvJe?utm_source=generator" width="100%" height="80" frameBorder="0" allowfullscreen="" allow="autoplay; clipboard-write; encrypted-media; fullscreen; picture-in-picture"></iframe>
+        `,
+        // backgroundColor: '#433e73'
+        backgroundColor: `linear-gradient(180deg, rgba(170,164,227,1) 0%, rgba(67,62,115,1) 37%, rgba(28,19,134,1) 100%)`
+    },
+    {
+        title: 'Jesus Is King',
+        dataName: '9',
+        description: [
+            "Every Hour", "Selah", "Follow God", "Closed on Sunday", "On God", "Everything We Need", "Water", "God Is", "Hands On", "Use This Gospel", "Jesus Is Lord"
+        ],
+        img: 'https://upload.wikimedia.org/wikipedia/en/thumb/a/a4/Kanye_West_-_Jesus_Is_King.png/220px-Kanye_West_-_Jesus_Is_King.png',
+        alt: 'JiK',
+        source: `
+            <iframe style="border-radius:12px" src="https://open.spotify.com/embed/album/0FgZKfoU2Br5sHOfvZKTI9?utm_source=generator" width="100%" height="80" frameBorder="0" allowfullscreen="" allow="autoplay; clipboard-write; encrypted-media; fullscreen; picture-in-picture"></iframe>
+        `,
+        // backgroundColor: '#2f1ee2'
+        backgroundColor: `linear-gradient(180deg, rgba(148,140,231,1) 0%, rgba(47,30,226,1) 37%, rgba(26,20,85,1) 100%)`
+    },
+    {
+        title: 'Donda',
+        dataName: '10',
+        description: [
+            "Donda Chant", "Jail", "God Breathed", "Off The Grid", "Hurricane", "Praise God", "Jonah", "Ok Ok", "Junya", "Believe What I Say", "24", "Remote Control",
+            "Moon", "Heaven and Hell", "Donda", "Keep My Spirit Alive", "Jesus Lord", "New Again", "Tell The Vision", "Lord I Need You", "Pure Souls", "Come to Life",
+            "No Child Left Behind", "Life Of The Party"
+        ],
+        img: 'https://upload.wikimedia.org/wikipedia/commons/thumb/a/a0/Almost_black_square_020305.png/640px-Almost_black_square_020305.png',
+        alt: 'Donda',
+        source: `
+            <iframe style="border-radius:12px" src="https://open.spotify.com/embed/album/5CnpZV3q5BcESefcB3WJmz?utm_source=generator" width="100%" height="80" frameBorder="0" allowfullscreen="" allow="autoplay; clipboard-write; encrypted-media; fullscreen; picture-in-picture"></iframe>
+        `,
+        // backgroundColor: 'black'
+        backgroundColor: `linear-gradient(180deg, rgba(0,0,0,1) 0%, rgba(0,0,0,1) 37%, rgba(0,0,0,1) 100%)`
+    }
+];
+
+arrOfDatas.forEach(item => {
+    renderAsideItems(item);
+});
+
+function renderAsideItems(item) {
+    aside.innerHTML += `
+        <button class='asideBtn' data-attribute='${item.dataName}'>${item.title}</button>
+    `;
+}
+
+let asideBtn = document.querySelectorAll('.asideBtn');
+asideBtn.forEach( el => {
+    el.addEventListener('click', renderMain);
+});
```

Comment:
Use [Event delegation](https://davidwalsh.name/event-delegate) for better performance and easier handing